### ISSUE TITLE
Can you share Train data set?

### ISSUE THREAD

**Issue by mini102** [aircarlo/cplx_bin2bin#1]
I tried Training with MSPLC DataSet (
1. Train : https://aecchallengepublic.blob.core.windows.net/plc2022/test_train.tar.gz in https://github.com/microsoft/PLC-Challenge
2. Val : https://aecchallengepublic.blob.core.windows.net/plc2022/blind.tar.gz)

My Process
a. Downloads Train dataset
b. batch size 16 -> 6 in config because of my GPU's ability.
c. python train_MSPLC.py

But, I got a bad result like this.

target:
![Image](https://github.com/user-attachments/assets/00b5f965-ff93-49f0-8386-0a0901619611)

input:
![Image](https://github.com/user-attachments/assets/1952ed5a-bcbb-43db-86ee-a5fc6a434c3f)

pred at 149 epoch:
![Image](https://github.com/user-attachments/assets/1808b9bd-c69c-4cfc-9a63-05328d1a5359)

So, I want to know how you can train and get good result like paper.

**Comment by UkiTenzai** [comment]
Hello, may I ask if you've found a pre-trained model? I need a model capable of handling packet loss longer than 1 second. To the best of my knowledge, only redundant coding can effectively address such cases—pure zero-prior speech inpainting can at most generate speech that sounds human-like but cannot recover the original meaningful content.

**Comment by drrrtata** [comment]
Hello, the output waveform of my pre-trained model is the same as yours. May I ask if this issue has been resolved now?
